src/lib/drivers/device/nuttx/SPI.cpp

```diff
@@ -169,6 +173,7 @@ SPI::_transfer(uint8_t *send, uint8_t *recv, unsigned len)
 	SPI_SETMODE(_dev, _mode);
 	SPI_SETBITS(_dev, 8);
 	SPI_SELECT(_dev, _device, true);
+	SPI_CS_TO_SCK_DELAY(_dev, _cs_to_sck_ns);
```

Comment:
SPI_SELECT, etc passed a dev are actual NuttX things, I would specifically not mirror the naming style/convention to make things clear.

Reply:
For at least one of the TII's custom platforms this will be implemented in the same way as SETBITS etc., i.e. inside NuttX SPI driver. In some platforms you can simply set the required delay from the transfer start to the first clock edge with a SPI IP register write. In that case there is no need for busylooping the delay.

So for that reason I do need the _dev there. My original intent was to suggest this to NuttX as well, but that is probably not needed.

So the macro name can be of course changed, would you have a suggestion for another name?

Reply:
> So the macro name can be of course changed, would you have a suggestion for another name?

Nothing specific, just minimize confusion with NuttX vs PX4 pieces.